resource-types/typescript-example-website-monitor/example-monitoring-website.json

```diff
@@ -102,6 +102,9 @@
         "/properties/Status",
         "/properties/SlaThreshold"
     ],
+    "writeOnlyProperties": [
+        "/properties/ApiKey"
```

Comment:
We may want to consider updating this to use `Type Configuration`.

Reply:
I see this [post](https://aws.amazon.com/blogs/mt/introducing-typescript-support-for-building-aws-cloudformation-resource-types/) references this resource with a walkthrough and also, as part of consuming the resource, I see the dynamic reference method that uses `ApiKey: "{{resolve:secretsmanager:typescript-example/apikey:SecretString}}"`; we might want to leave it as it is.